### ISSUE TITLE
training loss 

### ISSUE THREAD

**Issue by griffintin** [kujason/avod#28]
@kujason  
Hi, thank you for sharing the code. Ifollowed all the instructions and now training for CAR begins. Without modifying any code and using default config, training loss seems not right. It fluctuates and something loss are like this. Do you have any idea why this is happening? 

Step 500, Total Loss 2.645, Time Elapsed 6.096 s
Step 550, Total Loss 3.583, Time Elapsed 5.431 s
Step 560, Total Loss 11.722, Time Elapsed 5.333 s
Step 570, Total Loss 3.723, Time Elapsed 5.495 s
Step 580, Total Loss 1.895, Time Elapsed 5.473 s
Step 590, Total Loss 15.548, Time Elapsed 5.860 s
Step 600, Total Loss 2.417, Time Elapsed 5.647 s

**Comment by griffintin** [comment]
After 1 night trainning about 110000 iter., the loss greately decreased to about 0.x, without large fluctuation. Maybe the initial fluctuation is because of view drop?, defined as 
path_drop_probabilities: [0.9, 0.9]
